haystack/fields.py

```diff
@@ -298,7 +299,7 @@ def convert(self, value):
 
 
 class DecimalField(SearchField):
-    field_type = 'string'
+    field_type = 'float'
```

Comment:
I'm thinking that we should set type to `decimal` here so the backends have the option of using a decimal type if possible